<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1043</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="push_front">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>0</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>FrontView</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_top">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>30</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>TopView</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_side">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>60</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>SideView</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="OpenglWindow" name="openglWindow" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>861</width>
      <height>571</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Compute">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>140</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Compute</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1043</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOPENGL">
    <property name="title">
     <string>OPENGL</string>
    </property>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuFIle">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionSideView"/>
    <addaction name="actionFrontView"/>
    <addaction name="actionTopView"/>
    <addaction name="actionPerspectiveView"/>
   </widget>
   <addaction name="menuOPENGL"/>
   <addaction name="menuFIle"/>
   <addaction name="menuWindow"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSideView">
   <property name="text">
    <string>SideView</string>
   </property>
  </action>
  <action name="actionFrontView">
   <property name="text">
    <string>FrontView</string>
   </property>
  </action>
  <action name="actionTopView">
   <property name="text">
    <string>TopView</string>
   </property>
  </action>
  <action name="actionPerspectiveView">
   <property name="text">
    <string>PerspectiveView</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>OpenglWindow</class>
   <extends>QWidget</extends>
   <header location="global">openglwindow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
